src/core/mac/mac.cpp

```diff
@@ -800,8 +800,8 @@ void Mac::HandleReceiveTimer(void)
 void Mac::SentFrame(bool aAcked)
```

Comment:
Just a question. As Thread specifies: "Note that failures to transmit due to failed clear channel assessments do not count as a failure to receive an ACK."

If I correctly understand the code, function Mac::TransmitDoneTask will eventually call SentFrame(false) function when aError will be kThreadError_ChannelAccessFailure and max CSMA attempts have been done unsuccessfully. Which may cause neighbor to be removed due to high traffic in the network or jammer etc.

Probably it would be enough to pass ThreadError instead of bool to this function and act on it.